blog/_posts/2019-11-13-affiliations-feature.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: blog_post
+title: "Affiliation searches: the Why, What, and How of our Canonical Affiliations Feature"
+author: "Carolyn Stern Grant and Matthew Templeton"
+position: "ADS"
+category: blog
+label: general
+---
+
+One of the features new to ADS since the new interface has launched is search and organization by institutional affiliation.  The 15 million publications in ADS have more than 35 million combined author affiliations.  The ADS have long wanted to have these data in a searchable format, and we introduced this feature in early 2019.  Getting to this point took a lot of manual labor -- in fact, much more than the pipeline that resolves the affiliation strings -- and while this database will never be 100 percent complete, we’re hoping to use machine learning techniques to reduce the amount of human effort required to match affiliation data to their identifiers.
+
+We have said this before but it certainly bears repeating.  While affiliation information is largely complete for recent refereed literature, users need to remember that not every record contains an affiliation, therefore, searching by affiliation alone will inherently be incomplete.  We strongly recommend combining affiliation searches with author searches for best results.
```

Comment:
"We have said this before..." Where have we said it? Help pages, demos, user emails? Readers who haven't heard this before are going to tune out a bit. Maybe "We've mentioned this in our help pages and demonstrations of this feature to users, but it certainly bears repeating."